CMakeLists.txt

```diff
@@ -52,22 +56,13 @@ add_definitions("-DALL_INTERIOR_POINTERS -DNO_EXECUTE_PERMISSION")
 
 if (APPLE)
   if ("${CMAKE_OSX_ARCHITECTURES}" STREQUAL "")
-    set(CMAKE_OSX_ARCHITECTURES "ppc;i386;x86_64"
+    set(CMAKE_OSX_ARCHITECTURES "x86_64;i386"
```

Comment:
Does the presence of ppc lead to build failure on OS X?

Reply:
I'm not a Mac user myself, but this change enables Mac OS on **vcpkg**'s CI tests.

The following snippet causes a build failure when adding the `ppc` architecture:  

[`cdefs.h`](https://opensource.apple.com/source/xnu/xnu-4570.41.2/bsd/sys/cdefs.h.auto.html)
```
/*
 * Architecture validation for current SDK
 */
#if !defined(__sys_cdefs_arch_unknown__) && defined(__i386__)
#elif !defined(__sys_cdefs_arch_unknown__) && defined(__x86_64__)
#elif !defined(__sys_cdefs_arch_unknown__) && defined(__arm__)
#elif !defined(__sys_cdefs_arch_unknown__) && defined(__arm64__)
#else
#error Unsupported architecture
#endif
```

I assume that this only fails newer Mac OS versions.
I'm not sure in which version of MacOS was `ppc` support deprecated, but it is worth investigating so that we can conditionally pass the `ppc` architecture.

It is based on @nickwanniger 's fix for issue #275

Reply:
OK. It would be later to add arm and arm64 (when someone tests them)

Reply:
OK. It would be later to add arm and arm64 (when someone tests them).